src/utils/keys.js

```diff
@@ -1,21 +0,0 @@
-/*
-Copyright 2019 Adobe. All rights reserved.
-This file is licensed to you under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License. You may obtain a copy
-of the License at http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software distributed under
-the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
-OF ANY KIND, either express or implied. See the License for the specific language
-governing permissions and limitations under the License.
-*/
-
-/**
- * Returns an array whose items are the provided object's own enumerable
- * property keys.
- * @param {Object} obj
- * @returns {Array}
- */
-export default obj => {
-  return Object.keys(obj);
```

Comment:
Early in the Alloy project there were some conversations about what amount of complexity was required to be extracted into a utility. If I recall, the decision was that generally if it's a simple method call and no logic (conditions, etc) are involved, then it wouldn't qualify. If that's correct, I don't think this qualifies (although we do have a `noop` utility which probably would qualify when applying the same rules). Anyway, we were rarely using this utility and I've removed its usage.